### ISSUE TITLE
Look into izpack

### ISSUE THREAD

**Issue by zzak** [hone/mruby-cli#35]
I recently ran into [izpack](http://izpack.org/) and wonder if it can be used for packaging other kinds of applications, like for mruby-cli.

The source is available here: https://github.com/izpack/izpack

After a brief google, it seems like it can package just about anything:
http://www.infoq.com/news/2008/05/izpack_40

/cc @toch

**Comment by franckverrot** [comment]
Invoking @jponge, he might be able to help :-)

—

**Comment by jponge** [comment]
Yes it can package anything and create installers that "only" need a JVM to execute. 

That being said I haven't been involved in the project for many years, so you'd better ping the current team :-)

**Comment by zzak** [comment]
It seems ok if the JVM is only needed for the installer, but we wouldn't want it as a runtime dependency for CLI apps

**Comment by toch** [comment]
@zzak thanks, it was already on my radar as you can read on #28 . thanks! I'll try it.
